action.yml

```diff
@@ -27,6 +27,15 @@ inputs:
     default: "2.0.3"
     required: true
 
+  mongo-db:
+    description: "Whether to enable MongoDB instrumentation. Requires the MongoDB integration to be enabled on the organization."
+    required: false
```

Comment:
Change with `instruments`, a yaml array that could contain `mongodb`
Otherwise the key without `mongodb` would be simpler

Reply:
It is not possible to have a yaml array (see [here](https://stackoverflow.com/questions/75420197/how-to-use-array-input-for-a-custom-github-actions)), so I opted for a comma-separated list